hera_cal/abscal.py

```diff
@@ -1381,3 +1368,42 @@ def abscal_run(data_file, model_files, refant=None, calfits_infile=None, verbose
         return return_obj
 
 
+
+def _cut_bls(datacontainers, bls, bl_cut):
```

Comment:
Any reason this starts with an `_`?

Also, I'd recommend adding a unit test for this function.